### DISCUSSION TITLE
How does the bottom-left ToC like view works?

### DISCUSSION THREAD

**Discussion by KelSolaar** [zealdocs/zeal#1590] [Q&A] [answered]
Hello,

I have a documentation with `dashAnchor` elements and whilst they show fine in Dash, they don't in Zeal. I saw a few issues about implementing support for that but I was wondering how the current Zeal view that shows on some docs is working?

Cheers,

Thomas

**Comment by KelSolaar** [comment]
Found out: https://github.com/zealdocs/zeal/blob/9630cc94c155d87295e51b41fbab2bd5798f8229/src/libs/registry/docset.cpp#L340

One issue with this approach is that this does not work with docsets where the members are described in a child pages. Taking Unreal Engine for example, and its [FColorSpace](https://docs.unrealengine.com/5.3/en-US/API/Runtime/ColorManagement/FColorSpace/) class, the child pages are nested in sub-directories under the `FColorSpace` page:

![image](https://github.com/zealdocs/zeal/assets/99779/45f3c385-9726-4178-ac35-6e7a93f41989)

As a consequence, they are not listed.

**Reply by KelSolaar** [reply]
I will create an issue for this one!

**Reply by trollixx** [reply, marked as answer]
Tracked in #1591.
